channels_test.go

```diff
@@ -109,3 +109,89 @@ func TestSearchChannels(t *testing.T) {
 		}
 	}
 }
+
+func TestGetChannelInformation(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		statusCode    int
+		options       *Options
+		BroadcasterID string
+		respBody      string
+		parsed        []ChannelInformation
+	}{
+		{
+			http.StatusOK,
+			&Options{ClientID: "my-client-id"},
+			"44445592",
+			`{"data":[{"broadcaster_id":"44445592","broadcaster_login":"pokimane","broadcaster_name":"pokimane","broadcaster_language":"en","game_id":"509658","game_name":"Just Chatting","title":"See you Wednesday 8am for Among Us ^_^"}]}`,
+			[]ChannelInformation{
+				{
+					ID:         "44445592",
+					Display:    "pokimane",
+					Language:   "en",
+					CategoryID: "509658",
+					Category:   "Just Chatting",
```

Comment:
These will need to change also:

```suggestion
					BroadcasterID:         "44445592",
					BroadcasterName:    "pokimane",
					BroadcasterLanguage:   "en",
					GameID: "509658",
					GameName:   "Just Chatting",
```